SimplenoteTests/Verification/AccountVerificationRemoteTests.swift

```diff
@@ -5,25 +5,26 @@ import XCTest
 //
 class AccountVerificationRemoteTests: XCTestCase {
     private lazy var urlSession = MockURLSession()
-    private lazy var remote = AccountVerificationRemote(urlSession: urlSession)
+    private lazy var remote = AccountRemote(urlSession: urlSession)
 
     func testSuccessWhenStatusCodeIs2xx() {
         for _ in 0..<5 {
-            test(withStatusCode: Int.random(in: 200..<300), expectedResult: true)
+            test(withStatusCode: Int.random(in: 200..<300), expectedResult: Remote.Result.success)
         }
     }
 
     func testFailureWhenStatusCodeIs4xxOr5xx() {
         for _ in 0..<5 {
-            test(withStatusCode: Int.random(in: 400..<600), expectedResult: false)
+            let statusCode = Int.random(in: 400..<600)
+            test(withStatusCode: statusCode, expectedResult: Remote.Result.failure(statusCode, nil))
         }
     }
 
     func testFailureWhenNoResponse() {
-        test(withStatusCode: nil, expectedResult: false)
+        test(withStatusCode: nil, expectedResult: Remote.Result.failure(0, nil))
```

Comment:
By switching to Swift Result Type, this would end up looking like:

```
test(withStatusCode: nil, expectedResult: .failure(unknown))
```

With the detail that we should, probably, not allow for null errors.